src/services/contentful/contentful.ts

```diff
@@ -10,26 +12,36 @@ const contentfulClient = createClient({
   accessToken: REACT_APP_CONTENTFUL_TOKEN,
 });
 
-const config = {
+const config: ConfigProps = {
```

Comment:
The name can be just Config as typescript names are capitalised I don't think there will be confusion with this.